src/NuGet/BuildNuGets.csx

```diff
@@ -98,6 +99,7 @@ string[] RedistPackageNames = {
     "Microsoft.CodeAnalysis.Workspaces.Common",
     "Microsoft.VisualStudio.LanguageServices",
     "Microsoft.VisualStudio.LanguageServices.Next",
+    "Microsoft.VisualStudio.LanguageServices.Razor.RemoteClient",
```

Comment:
I added an edit and tagged @agocke to see if we can list these in `NonRedistPackageNames` and have things still work.